apollo-router/tests/integration/file_upload.rs

```diff
@@ -886,12 +887,12 @@ async fn it_fails_invalid_file_order() -> Result<(), BoxError> {
               },
               "errors": [
                 {
-                  "message": "HTTP fetch failed from 'uploads_clone': HTTP fetch failed from 'uploads_clone': error from user's HttpBody stream: error reading a body from connection: Missing files in the request: '1'.",
+                  "message": "HTTP fetch failed from 'uploads_clone': HTTP fetch failed from 'uploads_clone': client error (SendRequest)",
```

Comment:
hmmm this is missing the more important part of the error. Where did it go?

Reply:
This is one of the bad side-effects of the changes in Hyper regarding Body implementation. We still hope to resolve this either before the PR is merged or shortly after.